core/context.go

```diff
@@ -102,7 +104,7 @@ func (ctx *Context) Close() {
 	log.Info("shutting down IPVS context")
 
 	// This will also shutdown the pulse notification sink goroutine.
-	close(ctx.pulseCh)
+	close(ctx.stopCh)
```

Comment:
I don't get it. It was working fine already: `close(ctx.pulseCh)` will break the for loop in `notificationSink()`, why do we need another channel here?